taxonomy-editor-frontend/src/pages/gotoproject/index.jsx

```diff
@@ -65,7 +66,25 @@ const GotoProject = () => {
             { title: "Project", field: "projectName" },
             { title: "Taxonomy", field: "taxonomyName" },
             { title: "Branch", field: "branchName" },
-            { title: "Description", field: "description", width: "15vw" },
+            { title: "Description", field: "description", width: "10vw" },
+            {
+              title: "Errors",
+              field: "errors_count",
+              render: (rowData) => {
+                return rowData["errors_count"] > 0 ? (
+                  <MuiLink
+                    color="error"
+                    href={`/${toSnakeCase(rowData["taxonomyName"])}/${
+                      rowData["branchName"]
+                    }/errors`}
+                  >
+                    {rowData["errors_count"] + " errors"}
+                  </MuiLink>
+                ) : (
+                  <Typography color="success">✅</Typography>
```

Comment:
think is ok to leave it empty if no error

Reply:
No feedback seemed a little weird, so I added a hyphen here